### DISCUSSION TITLE
multiple secrets in the same line not detected properly

### DISCUSSION THREAD

**Discussion by MorAlon1** [aquasecurity/trivy#4333] [Bugs]
When single string or line contains two different secrets identified by two different rules, trivy's output includes the detection of both of the secrets and the rules names, but masks only the first secret in both. 

please take a look at the attached image (the second secret suppose to be "user=***")
<img width="774" alt="Screen Shot 2023-03-15 at 15 59 50" src="https://user-images.githubusercontent.com/101275199/225335905-29c15ff7-da32-42db-90bb-b2eb4b28195d.png">

**Comment by knqyf263** [comment]
>the second secret suppose to be "user=***"

Do you mean `User=True` should be detected?
